manifests/install.pp

```diff
@@ -6,14 +6,30 @@
   ensure_packages($::oxidized::install_dependencies, { 'before' => 'Package[oxidized]' })
 
   if $facts.dig('os', 'family') == 'RedHat' {
-    $provider = 'scl_gem'
-    file { '/usr/local/bin/scl_gem':
-      ensure => 'file',
-      owner  => 'root',
-      group  => 'root',
-      mode   => '0755',
-      source => 'puppet:///modules/oxidized/scl_gem',
-      before => Package['oxidized'],
+    if versioncmp($::operatingsystemrelease, '8') >= 0 {
```

Comment:
Be good to use facts hash here, `$facts['os']['release']['major']` if this condition is still needed.